src/main/java/org/assertj/core/configuration/ConfigurationProvider.java

```diff
@@ -27,6 +30,8 @@
  * @author Filip Hrisafov
  * @since 2.9.0 / 3.9.0
  */
+@ServiceConsumer(value = Configuration.class, resolution = OPTIONAL)
+@ServiceConsumer(value = Representation.class, resolution = OPTIONAL)
```

Comment:
What problem does putting the service optional solves?

I'm asking the question because we prefer having no additional dependencies.

Reply:
You know what? The service loader change is frivolous and takes focus ways from the verification issue so I just remove that commit. Please take another look.